generators/server/templates/src/main/java/package/config/DatabaseConfiguration.java.ejs

```diff
@@ -27,6 +27,7 @@ import liquibase.integration.spring.SpringLiquibase;
 <%_ if (databaseType === 'mongodb') { _%>
 import com.github.mongobee.Mongobee;
 import com.mongodb.MongoClient;
+import org.springframework.data.mongodb.repository.MongoRepository;
```

Comment:
I think this is only when you also use Elasticsearch - I'll have a look and correct if necessary